src/d3a/blockchain/utils.py

```diff
@@ -50,8 +50,8 @@ def unlock_account(chain, address):
     if ConstSettings.BlockchainSettings.START_LOCAL_CHAIN:
         return
     else:
+        account_pass = os.getenv("BLOCKCHAIN_ACCOUNT_PASSWORD", "testgsy")
```

Comment:
So I understand, that the user has to set the BLOCKCHAIN_ACCOUNT_PASSWORD in an environmental variable, right? If not done already, please add it to the Wiki. Thanks!

Reply:
Will do, thanks!